### PR TITLE
Refreshed integrate wallet into UI

### PR DESCRIPTION
**Pull request by davidyuk**
Rebased and fixed version of #528 

depends on #720 https://github.com/aeternity/superhero-wallet/pull/423 https://github.com/aeternity/aepp-governance/pull/147 https://github.com/aeternity/aepp-sdk-js/pull/1062

### CODE REVIEW

**Comment by thepiwo** [comment]
works for me, UI/UX should be improved, just a few points:

- alert popups connection in browser
- confirmation dialog in wallet to big in frame
- for e.g. retip it hides the retip dialog confirming the tx
- confirmation not really prominent in UI, maybe black out all except wallet with 50% opacity if confirmation needed

**Comment by davidyuk** [comment]
It looks like that the latest Chrome in Incognito mode disallows access to `localStorage` in iframes from other domains, this breaks integrated wallet with this exception:

> Uncaught (in promise) DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.

**Comment by mradkov** [comment]
> works for me, UI/UX should be improved, just a few points:
> 
>     * alert popups connection in browser
> 
>     * confirmation dialog in wallet to big in frame
> 
>     * for e.g. retip it hides the retip dialog confirming the tx
> 
>     * confirmation not really prominent in UI, maybe black out all except wallet with 50% opacity if confirmation needed

@davidyuk can you address these?

**Comment by davidyuk** [comment]
Implementation of https://github.com/aeternity/jitsi-meet/issues/122 should reduce the number of edge cases that breaks communication with the wallet.

File: .env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-VUE_APP_WALLET_URL=https://wallet.superhero.com
+VUE_APP_WALLET_URL=https://develop.wallet.z52da5wt.xyz/
```
Comment (mradkov) [inline .env:1]: Switch this before release. 💭

**Pull request merged**